package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "multi-calendar-dates",
```

Comment:
We need to prefix with the `@dhis2` namespace here, so `@dhis2/multi-calendar-dates`

Reply:
FYI: this is not in-scope for this PR, but will need to be changed